site/components/pages/Home/index.module.css

```
@@ -107,6 +107,17 @@
   gap: 1.6rem;
 }
 
+.lowerStack {
+  display: flex;
+  gap: 1rem;
```

Designers use an 8 pixel grid, so I try to stick to that, or sometimes use increments of 4px (0.4rem)
so lets make this 1.6rem
combined with my other suggestion below it lines up nicely!
```suggestion
  gap: 1.6rem;
```